crates/script/src/transaction.rs

```diff
@@ -66,31 +66,6 @@ impl TransactionWithMetadata {
         Self { transaction: WithOtherFields::new(transaction), ..Default::default() }
     }
 
-    pub fn from_zk_tx_request(transaction: TransactionRequest, zk: Option<ZkTransaction>) -> Self {
-        Self { transaction: WithOtherFields::new(transaction), zk, ..Default::default() }
-    }
-
-    pub fn new(
```

Comment:
If we are gonna remove this, doesn't it make more sense to then rename `new_with_zk` to `new`?